index.html

```diff
@@ -2,11 +2,32 @@
 <html>
 <head>
 	<title>JS Onboard Project</title>
+
 	<script type="text/javascript" charset="utf-8" src="third_party/jquery-2.0.3.min.js"></script>
+	<script src="app.js"> </script>
+	<link rel="stylesheet" type="text/css" href="/style.css">
 </head>
 
-<body>
-	<p>Hello</p>
+<body id="body">
+	<div id="form-content">
+			Go To Start Index:<input type="number" id="index" value="0">
```

Comment:
You should add a `min` here. to prevent the user from inputting negative numbers.